### ISSUE TITLE
Block Documentation - Dead link to Block Components List

### ISSUE THREAD

**Issue by jamiechapman** [MicrosoftDocs/minecraft-creator#924]
**Describe the missing content**
The `components` parameter has a link to an invalid page. The "Block Components List" text links to a 404 page. 

**Vanilla usage example**
N/A

**Additional context**
- Page where the URL is linked from: [Block Documentation - Block JSON File Structure](https://learn.microsoft.com/en-us/minecraft/creator/reference/content/blockreference/examples/blockjsonfilestructure?view=minecraft-bedrock-stable)
- Invalid URL: [Block Components List](https://learn.microsoft.com/en-us/minecraft/creator/reference/content/blockreference/examples/%5CBlockComponents%5CBlockComponentsList.md?view=minecraft-bedrock-stable)

**Comment by iconicNurdle** [comment]
Good catch! Thank you. Link is fixed and will be apparent after the next document push.
